site/src/sources/kolibri.json

```diff
@@ -23,26 +23,13 @@
       ]
     },
     {
-      "name": "Accordion",
-      "url": "https://public-ui.github.io/docs/components/accordion",
-      "definition": "Die **Accordion**-Komponente ist ein Aufklapp-Menü. Klickt man auf den Kopfbereich, bestehend aus Icon und Überschrift, öffnet sich ein Textfeld mit zusätzlichen Informationen. Somit ist es ein interaktives Navigationselement, welches dazu dient, umfangreiche Inhalte platzsparend darzustellen.\n\nAccordions kommen immer dann zum Einsatz, wenn einem thematischen Oberbegriff zugeordnete Inhalte angezeigt oder verborgen werden sollen. Sie erlauben umfangreichere Detailinformationen zu einem Oberbegriff, als es aus Gründen der Übersichtlichkeit eigentlich sinnvoll wäre. Sie überlassen es den Besucher:innen selbst, ob sie sich diese Informationen anzeigen lassen möchten.",
+      "name": "accordion",
```

Comment:
So changing this from "Accordion" to "accordion" means that it no longer appears in the list of widget names by design system; compare these pages:
https://open-ui.org/components/accordion.research/
https://deploy-preview-750--open-ui.netlify.app/components/accordion.research/

This could be solved by adding an 'openUIName' field saying 'Accordion', but then you'd have something that looks like this in the Names section:
https://open-ui.org/components/disclosure.research/
except with one line for "Accordion" and one line for "accordion".

(I think that's why I changed "Accordion" to uppercase at some point in the past, because I was working on the page.)

I think it would be better to uppercase all of the "name" fields here.  (And a later PR might also add appropriate `openUIName` fields to make the widgets show up on the relevant research pages when that isn't sufficient.)

Reply:
I have add the `openUiName` to all our meta informations in the `kolibri.json`. I hope it was the right way. Please let me know, if I should do something else.